README.md

```diff
@@ -353,6 +353,34 @@ as follows:
 }
 ```
 
+If you are not using NixOS you can place the following flake in ``~/.config/nixpkgs/flake.nix``
+to load your standard home-manager config:
+
+```nix
+{
+  description = "A Home-Manager flake";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+    home-manager.url = "github:nix-community/home-manager";
+  };
+
+  outputs = inputs: {
+    homeConfigurations = {
+      jdoe = inputs.home-manager.lib.homeManagerConfiguration rec {
+        system = "x86_64-linux";
+        homeDirectory = "/home/${username}";
```

Comment:
Might be nicer to simply set the `homeDirectory` explicitly:

```suggestion
      jdoe = inputs.home-manager.lib.homeManagerConfiguration {
        system = "x86_64-linux";
        homeDirectory = "/home/jdoe";
```

also allows the `rec` to be removed.